unfoldingWord/translationCore
# fixed builds CI support

#### This pull request addresses:

* Long wait times for macOS builds.
* Failing Linux builds
* Failing Windows builds

See #2417.

#### How to test this pull request:

1. Make sure Travis tests run successfully on this PR.
2. Merge in this PR
3. Watch for successful builds on the `develop` branch in Travis.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2708)
